Source/DataSources/KmlDataSource.js

```diff
@@ -1357,7 +1502,7 @@ define([
             var length = childNodes.length;
             for (var q = 0; q < length; q++) {
                 var child = childNodes[q];
-                if (child.localName === featureName && namespaces.kml.indexOf(child.namespaceURI) !== -1) {
+                if (child.localName === featureName) {
```

Comment:
This original code just ignores non-kml namespace feature (eg. `gx:Tour`). Now that we want to print out warning messages, I added `Tour` to `featureTypes` so it calls `processUnsupported` just like `ScreenOverlay` and `PhotoOverlay`.

Was there ever any documents where people used other namespaces with localNames the same as standard features like `<cesium:Placemark>` that we just want to ignore?